public/locales/my/translation.json

```
@@ -98,6 +98,20 @@
   "Schedule Export": "အချိန်ဇယားကို ထုတ်ယူရန်",
   "Generate a high-fidelity cryptographic ledger of your current courses.": "သင်၏လက်ရှိဘာသာရပ်များ၏ မှတ်တမ်းကို ထုတ်ယူပါ။",
   "SYCHRONIZING...": "တစ်ပြိုင်တည်းလုပ်ဆောင်နေသည်...",
+  "General": "အထွေထွေ",
```

Biome (2.4.13)

[error] The key General was already declared.

(lint/suspicious/noDuplicateObjectKeys)